### DISCUSSION TITLE
Patch banks and TGs

### DISCUSSION THREAD

**Discussion by pinkmouse101** [probonopd/MiniDexed#337] [General]
Hi folks, just pondering whilst i work out what I want to do in the physical/electronics build, but is there a way of setting different TGs to use different sound banks? Say, TG1 and 2 use the stock factory set, 3 and 4 use my own modded set, and the other two pairs use a couple of different downloaded ones. Might be handy...

cheers
al

**Comment by diyelectromusic** [comment]
I could be wrong but I think you can set the sound bank in the performance.ini file for each TG, but all still have access to the full set I think? That just sets the initial sound bank of course, you can still change them all from the menus or using MIDI bank select messages, but then you'd need different TGs on different MIDI channels of course.

**Reply by pinkmouse101** [reply]
Thanks Kevin, I think I need to go through this all much more thoroughly, and perhaps RTFM! I just get distracted by interesting sounds... :-D
